DefensiveItem.h

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
Don't include what you don't use.
Take a look at the previous lines and determine if you need to include these files
- [ ] revise